hedera-node/hedera-app/src/main/java/com/hedera/node/app/authorization/PrivilegesVerifier.java

```diff
@@ -221,4 +247,11 @@ private SystemPrivilege checkCryptoUpdate(
     private SystemPrivilege checkEntityDelete(final long entityNum) {
         return isSystemEntity(entityNum) ? IMPERMISSIBLE : UNNECESSARY;
     }
+
+    // move to somewhere else
```

Comment:
Should this be moved elsewhere per the comment?

Reply:
We should just delete this entire block imo @lukelee-sl , c.f. https://github.com/hashgraph/hedera-services/issues/9383